rq/registry.py

```diff
@@ -83,14 +83,24 @@ def __contains__(self, item: Union[str, 'Job']) -> bool:
 
     @property
     def count(self) -> int:
-        """Returns the number of jobs in this registry
+        """Returns the number of jobs in this registry and runs cleanup
 
         Returns:
             int: _description_
         """
         self.cleanup()
         return self.connection.zcard(self.key)
 
+    def get_job_count(self) -> int:
```

Comment:
Mind adding a `cleanup=True` argument here as well? As a rule of thumb, I want RQ to return the correct answer to everyone so it's beginner proof. But should you need more performance, it's available for you.

Another alternative is that we make this an internal API (`_get_job_count()`) and just point users to use `registry.count`

Reply:
@selwin I've added the `cleanup=True` parameter; rebased on top of the current master, so you could merge cleanly; made `count` call `get_job_count`, to decrease the chance of desyncronization of the code branches.

Let me know if you need anything else.

Reply:
@selwin Is there any plan with regards to when those changes might be released?

Reply:
@selwin Disregard my question about the release--I have just saw your answer in another issue thread.